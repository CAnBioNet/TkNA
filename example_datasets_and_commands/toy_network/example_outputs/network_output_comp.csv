Edge name,partner1,partner2,Edge Type,Pearson corr pval (Expt 1),Pearson corr pval (Expt 2),Combined corr Pvalue,rho Coefficient (Expt 1),rho Coefficient (Expt 2),Mean rho Coefficient,Combined corr FDR,partner1 name,partner1_MedianValue (Expt 1),partner1_MedianValue (Expt 2),partner1_MeanValue (Expt 1),partner1_MeanValue (Expt 2),partner1_MedianLog2FoldChange (Expt 1),partner1_MedianLog2FoldChange (Expt 2),partner1_MeanLog2FoldChange (Expt 1),partner1_MeanLog2FoldChange (Expt 2),partner2 name,partner2_MedianValue (Expt 1),partner2_MedianValue (Expt 2),partner2_MeanValue (Expt 1),partner2_MeanValue (Expt 2),partner2_MedianLog2FoldChange (Expt 1),partner2_MedianLog2FoldChange (Expt 2),partner2_MeanLog2FoldChange (Expt 1),partner2_MeanLog2FoldChange (Expt 2),Correlations Passed Consistency Filter (All Agree),All Non-PUC Filters Passed,combined Coefficient correlation Direction,partner1_FC_direction,partner2_FC_direction,IfFoldChangeDirectionMatch,PUC,"Final Network Value (0: No edge, 1: Positive edge, -1: Negative edge)"
phenoC<==>phenoD,phenoC,phenoD,pheno<==>pheno,0.0007234970415914797,0.0007234970415914797,8.093986020216944e-06,0.8824370576924025,0.8824370576924025,0.8824370576924025,4.8563916121301665e-05,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,True,1.0,-1.0,-1.0,1.0,1,1.0
phenoC<==>phenoB,phenoC,phenoB,pheno<==>pheno,0.0641703684849123,0.0641703684849123,0.026734752723508733,-0.6044458181469112,-0.6044458181469112,-0.6044458181469112,0.053469505447017465,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,True,True,-1.0,-1.0,1.0,-1.0,1,-1.0
phenoB<==>phenoD,phenoB,phenoD,pheno<==>pheno,0.011356130222475556,0.011356130222475556,0.0012839419958379935,-0.7563271513870781,-0.7563271513870781,-0.7563271513870781,0.0038518259875139804,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
geneF<==>geneE,geneF,geneE,gene<==>gene,0.23106424314547172,0.23106424314547172,0.20983174323611303,-0.4165976673276287,-0.4165976673276287,-0.4165976673276287,0.6294952297083392,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,True,-1.0,-1.0,1.0,-1.0,1,-1.0
geneG<==>geneE,geneG,geneE,gene<==>gene,0.9614423092252469,0.9614423092252469,0.9970651988323839,0.0176318533335005,0.0176318533335005,0.0176318533335005,0.9970651988323839,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,True,1.0,1.0,1.0,1.0,1,1.0
geneE<==>phenoA,geneE,phenoA,gene<==>pheno,0.08642203736592723,0.08642203736592723,0.04404351591344755,0.5684455204105898,0.5684455204105898,0.5684455204105898,0.34772866492141546,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,True,True,1.0,1.0,1.0,1.0,1,1.0
phenoB<==>phenoA,phenoB,phenoA,pheno<==>pheno,0.07923133740558574,0.07923133740558574,0.03810987476820658,0.5793429920898184,0.5793429920898184,0.5793429920898184,0.05716481215230987,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,True,True,1.0,1.0,1.0,1.0,1,1.0
geneG<==>geneF,geneG,geneF,gene<==>gene,0.7161975981018766,0.7161975981018766,0.8553762293570769,-0.13201503628139225,-0.13201503628139225,-0.13201503628139225,0.9970651988323839,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
