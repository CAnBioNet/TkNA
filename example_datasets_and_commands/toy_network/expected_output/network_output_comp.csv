Edge name,partner1,partner2,Edge Type,Pearson corr pval (Expt 1),Pearson corr pval (Expt 2),Combined corr Pvalue,Corr Coefficient (Expt 1),Corr Coefficient (Expt 2),Mean Corr Coefficient,Combined corr FDR,partner1 name,partner1_MedianValue (Expt 1),partner1_MedianValue (Expt 2),partner1_MeanValue (Expt 1),partner1_MeanValue (Expt 2),partner1_MedianLog2FoldChange (Expt 1),partner1_MedianLog2FoldChange (Expt 2),partner1_MeanLog2FoldChange (Expt 1),partner1_MeanLog2FoldChange (Expt 2),partner2 name,partner2_MedianValue (Expt 1),partner2_MedianValue (Expt 2),partner2_MeanValue (Expt 1),partner2_MeanValue (Expt 2),partner2_MedianLog2FoldChange (Expt 1),partner2_MedianLog2FoldChange (Expt 2),partner2_MeanLog2FoldChange (Expt 1),partner2_MeanLog2FoldChange (Expt 2),Correlations Passed Consistency Filter (All Agree),All Non-PUC Filters Passed,combined Coefficient correlation Direction,partner1_FC_direction,partner2_FC_direction,IfFoldChangeDirectionMatch,PUC,"Final Network Value (0: No edge, 1: Positive edge, -1: Negative edge)"
phenoC<==>phenoD,phenoC,phenoD,pheno<==>pheno,0.0007234970415914797,0.0007234970415914797,8.093986020216944e-06,0.8824370576924025,0.8824370576924025,0.8824370576924025,4.8563916121301665e-05,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.3219280948982743,-3.3219280948982743,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,True,1.0,-1.0,-1.0,1.0,1,1.0
phenoB<==>phenoD,phenoB,phenoD,pheno<==>pheno,0.01135613022247554,0.01135613022247554,0.0012839419958379904,-0.7563271513870781,-0.7563271513870781,-0.7563271513870781,0.0038518259875139713,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
geneG<==>geneF,geneG,geneF,gene<==>gene,0.7161975981018764,0.7161975981018764,0.8553762293570768,-0.13201503628139225,-0.13201503628139225,-0.13201503628139225,0.9970651988323839,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
phenoB<==>phenoC,phenoB,phenoC,pheno<==>pheno,0.0641703684849123,0.0641703684849123,0.026734752723508733,-0.6044458181469112,-0.6044458181469112,-0.6044458181469112,0.053469505447017465,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.3219280948982743,-3.3219280948982743,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
geneG<==>geneE,geneG,geneE,gene<==>gene,0.9614423092252463,0.9614423092252463,0.9970651988323839,0.0176318533335005,0.0176318533335005,0.0176318533335005,0.9970651988323839,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,True,1.0,1.0,1.0,1.0,1,1.0
geneE<==>geneF,geneE,geneF,gene<==>gene,0.23106424314547197,0.23106424314547197,0.20983174323611328,-0.4165976673276287,-0.4165976673276287,-0.4165976673276287,0.6294952297083399,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
phenoA<==>geneE,phenoA,geneE,pheno<==>gene,0.08642203736592723,0.08642203736592723,0.04404351591344755,0.5684455204105898,0.5684455204105898,0.5684455204105898,0.3477286649214157,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,True,1.0,1.0,1.0,1.0,1,1.0
phenoA<==>phenoB,phenoA,phenoB,pheno<==>pheno,0.0792313374055858,0.0792313374055858,0.03810987476820663,0.5793429920898184,0.5793429920898184,0.5793429920898184,0.05716481215230995,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,True,True,1.0,1.0,1.0,1.0,1,1.0
